regen_openapi.sh

```diff
@@ -17,3 +17,5 @@ yarn openapi-generator-cli generate -i openapi.json -g java -o java/lib/generate
 yarn openapi-generator-cli generate -i openapi.json -g kotlin -o kotlin/lib/generated/openapi -c kotlin/openapi-generator-config.json -t kotlin/templates
 
 yarn openapi-generator-cli generate -i openapi.json -g ruby -o ruby -c ruby/openapi-generator-config.json -t ruby/templates
+
```

Comment:
I like moving each one of these to its own yarn command, honestly can just remove this shell script in favor of your new approach to avoid duplication

Reply:
Sounds great!